etc/requirements.txt

```diff
@@ -2,9 +2,8 @@
 configparser==3.5.0
 enum34==1.1.6
 flake8==3.7.9
-nose==1.3.7
-nose-ignore-docstring==0.2
-nose-timer==0.7.5
+pytest==4.6.11;python_version<"3.5"
+pytest==6.0.1;python_version>"2.7"
```

Comment:
Am I right that this file is meant to be a lockfile? If so, then we should pin the transitive dependencies too.

Reply:
Not really sure. This is a relic from the appveyor/travis days that we seem to have never touched. I can convert this to a pip-tools flow, but I'd like to do that as a separate PR if that's okay.

Reply:
Sounds good.

Reply:
https://github.com/quantopian/trading_calendars/issues/181